Sources/StreamChatUI/ChatMessageList/ChatMessageListScrollOverlayView.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// View that is displayed as top overlay when message list is scrolling
+open class ChatMessageListScrollOverlayView: _View, AppearanceProvider {
+    open var text: String? {
+        didSet { updateContentIfNeeded() }
+    }
```

Comment:
nit: this should ideally be called `content` (because of automatic SwiftUI wrapper export)